explorer/client/src/pages/object-result/ObjectResult.tsx

```diff
@@ -39,15 +43,44 @@ const Fail = ({ objID }: { objID: string | undefined }): JSX.Element => {
     );
 };
 
+// Get the data for the object ID and address that publishes a Package
+function getObjectDataWithPackageAddress(objID: string, network: string) {
+    return rpc(network)
+        .getObject(objID as string)
+        .then((objState) => {
+            const resp: DataType = translate(objState) as DataType;
+            if (resp.objType === 'Move Package' && resp.data.tx_digest) {
+                return rpc(network)
+                    .getTransactionWithEffects(resp.data.tx_digest)
+                    .then((txEff: TransactionEffectsResponse) => ({
+                        ...resp,
+                        publisherAddress: getTransactionSender(
+                            txEff.certificate
+                        ),
+                    }))
+                    .catch((err) => {
+                        console.log(err);
+                        // TODO: Not sure if I should show Genesis as Package Publisher or ignore it
```

Comment:
I like the overall approach; it's efficient. 

Regarding packages such as 0x2 that exist at Genesis, perhaps the best approach would be to remove the row detailing the `Last Transaction ID` and input 'Genesis' as a string next to `Publisher`? Would this be a better approach, Bauer (@bwann52)? 

Below is a screenshot of what this PR delivers for package 0x2:

![image](https://user-images.githubusercontent.com/11377188/175573350-01580c03-d163-4646-8e71-338f6e198146.png)

Note that the blue Genesis text is clickable and will take you to the following error page:

![image](https://user-images.githubusercontent.com/11377188/175573529-2083de9c-9a1e-4205-bc19-84af06d04ca3.png)

A possible alternative would be the following, with no clickable link:
![image](https://user-images.githubusercontent.com/11377188/175574108-28e6928f-9fc1-4bad-91d0-561276d74737.png)